README.md

```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
+| [Interspeech 2018](http://www.interspeech2018.org/index.html) | 2-6 Sep | Hyderabad International Convention Centre | Interspeech is the world’s largest and most comprehensive conference on the science and technology of spoken language processing |
+| [DevOps India Summit](https://www.xellentro.com/devops-india-summit-2018/) | 31 Aug - 1 Sep | Bengaluru Marriott Hotel Whitefield | DevOps India Summit 2018 is a conference for the Indian DevOps Community to hear for the first time the stalwarts of DevOps on what and how they have implemented DevOps in their complex organization and also know what the world is doing today in DevOps. |
 | [ODSC (One Data Science Conference) ](https://india.odsc.com/) | 30 Aug – 2 Sep | Hotel Lalit Ashok, Bengaluru  | Conference on AI and Data Science |
 | [ReactFoo-Delhi](https://reactfoo.in/2018-delhi/) | 18 Aug | TBA | React, alternatives to React, React Native and front-end engineering. |
 | [DevConf.in 2018](https://devconf.info/in) | 4-5 Aug | Christ University - Bengaluru | Annual Developers' Conference organised by Red Hat for local FOSS community |
 | [Deccan RubyConf 2018](https://www.deccanrubyconf.org/) | 4 Aug | Sheraton Grand Hotel, Pune | Pune extension of RubyConf India organized by same folks organizing RubyConf India |
+| [Blockchain Summit](http://www.unicomlearning.com/Blockchain_Summit_Mumbai-2018/) | 3 Aug | Mumbai | This conference brings together Block Chain technologists in the finance sector, regulators, industry commentators, computer science researchers and others involved in financial innovation; these experts explore some of the technology and the many applications of Block chain in Financial Services. |
 | [Container Conference](https://www.containerconf.in/) | 3-4 Aug | Radisson Blu (Park Plaza), ORR, Bangalore | Container Conference is a developer focused conference on Docker, Managed Kubernetes/Container Services, Service Mesh, Kubernetes |
+| [Data Visualisation Summit](http://www.unicomlearning.com/Data_Visualisation_Summit_Mumbai_2018/) | 2 Aug | The LaLiT, Mumbai | A one-day, single-track event focused exclusively contextualized to explore Visualisation tools and tips. Bringing together data enthusiasts who will help you rethink your data imagery to enable easier to interpret, management and action. |
+| [Call For Code Day](http://callforcodeindia.com/) | 20 Jul | BHAU Institute, Pune | Call for Code is more than a global challenge, it's an opportunity for developers to make a lasting change for social good. Unlike other hackathons and code challenges, IBM is providing developers with access to the best in IBM technology - Cloud, AI, IoT and more, to create applications that improve disaster preparedness, build resilient communities and safeguard the health and well-being of individuals and institutions. |
```

Comment:
Please remove all instances of this conference since all of them do not have a CFP.